lisa/tests/test_testsuite.py

```diff
@@ -227,39 +227,59 @@ def test_test_result_canrun(self) -> None:
     def test_skip_before_suite_failed(self) -> None:
         test_suite = self.generate_suite_instance()
         test_suite.set_fail_phase(fail_on_before_suite=True)
-        test_suite.start(environment=self.default_env, case_results=self.case_results)
+        test_suite.start(
+            environment=self.default_env,
+            case_results=self.case_results,
+            case_variables={},
+        )
         for result in self.case_results:
             self.assertEqual(TestStatus.SKIPPED, result.status)
             self.assertEqual("before_suite: failed", result.message)
 
     def test_pass_after_suite_failed(self) -> None:
         test_suite = self.generate_suite_instance()
         test_suite.set_fail_phase(fail_on_after_suite=True)
-        test_suite.start(environment=self.default_env, case_results=self.case_results)
+        test_suite.start(
+            environment=self.default_env,
+            case_results=self.case_results,
+            case_variables={},
```

Comment:
Can we get a (testing) test assert that it gets some of these?

Reply:
added